weblate/trans/views/create.py

```diff
@@ -520,10 +520,20 @@ def form_valid(self, form):
             return self.redirect_create(
                 repo=component.get_repo_link_url(),
                 project=component.project.pk,
+                category=component.category.pk,
                 name=form.cleaned_data["name"],
                 slug=form.cleaned_data["slug"],
                 vcs=component.vcs,
                 source_language=component.source_language.pk,
+                license=component.license,
+                agreement=component.agreement,
+                merge_style=component.merge_style,
+                commit_message=component.commit_message,
+                add_message=component.add_message,
+                delete_message=component.delete_message,
+                merge_message=component.merge_message,
+                addon_message=component.addon_message,
+                pull_message=component.pull_message,
```

Comment:
While you include these in the URL, there is nothing to process these (`ComponentCreateForm` doesn't accept these). The additional fields either should be hidden inputs in the create form, or it should have tabs/folding for the additional parameters.

Reply:
We can keep the fields as hidden inputs in create form.